packages/flab/stringify.js

```diff
@@ -0,0 +1,9 @@
+console.log(
+	'module.exports = '+
+	require("fs").readFileSync("/dev/stdin", "utf-8")
```

Comment:
I feel like I should know this, but... do we officially support windows? Seems like `/dev/stdin` wouldn't work there?